flashinfer/fused_moe/core.py

```
@@ -969,6 +1036,28 @@ def cutlass_fused_moe(
         Optional ``[gemm1_profile, gemm2_profile]`` override. Both values are absolute indices in
         the runner's combined tactic list; ``-1`` keeps the default tactic for that GEMM.
 
+    workspace_buffer : Optional[torch.Tensor]
+        Pre-allocated scratch buffer reused across calls to eliminate per-call workspace
+        allocation (which can reach 10–20 GiB at large batch size). If ``None`` (default),
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)